Upstreamed at: https://github.com/rizinorg/rizin/pull/3380

commit e1a65044139443c43a14c8304d095d9d6cddae56
Author: John Helmert III <ajak@gentoo.org>
Date:   Sat Feb 18 18:22:42 2023 -0600

    fix capstone include paths
    
    capstone include paths were adjusted in
    2b8104bc5e763ed841d6dbffacbeaf02e86b8421, and this wasn't picked up in
    the "PPC uplifting" commit, leading to a build failure with system
    capstone.
    
    Also fix the path of the ppc.h include, which seems to mistakenly use
    capstone's ppc.h when building with bundled capstone.
    
    Fixes: b46e7bd4f2 ("PPC uplifting to RzIL (#2823)")
    Signed-off-by: John Helmert III <ajak@gentoo.org>

diff --git a/librz/analysis/arch/ppc/ppc_il.h b/librz/analysis/arch/ppc/ppc_il.h
index ee91eb050b..7871414cef 100644
--- a/librz/analysis/arch/ppc/ppc_il.h
+++ b/librz/analysis/arch/ppc/ppc_il.h
@@ -9,7 +9,7 @@
 #include <rz_analysis.h>
 #include <rz_il.h>
 #include <rz_types.h>
-#include <capstone.h>
+#include <capstone/capstone.h>
 
 #define PPC_BYTE  8
 #define PPC_HWORD 16
diff --git a/librz/analysis/arch/ppc/ppc_il_ops.c b/librz/analysis/arch/ppc/ppc_il_ops.c
index f7a2261093..31a730c009 100644
--- a/librz/analysis/arch/ppc/ppc_il_ops.c
+++ b/librz/analysis/arch/ppc/ppc_il_ops.c
@@ -1,13 +1,13 @@
 // SPDX-FileCopyrightText: 2022 Rot127 <unisono@quyllur.org>
 // SPDX-License-Identifier: LGPL-3.0-only
 
-#include "ppc.h"
+#include "opcode/ppc.h"
 #include "ppc_il.h"
 #include "ppc_analysis.h"
 #include "rz_types_base.h"
 #include <rz_il/rz_il_opcodes.h>
 #include <rz_util/rz_assert.h>
-#include <capstone.h>
+#include <capstone/capstone.h>
 #include <rz_il/rz_il_opbuilder_begin.h>
 
 static RzILOpEffect *load_op(RZ_BORROW csh handle, RZ_BORROW cs_insn *insn, const cs_mode mode) {
